client/verta/verta/endpoint/build/_build.py

```diff
@@ -11,6 +11,11 @@ class Build:
     """
     An initiated docker build process of a deployable model.
 
+    .. versionchanged:: 0.23.0
+        Moved from ``verta.endpoint.Build`` to ``verta.endpoint.build.Build``.
+    .. versionadded:: 0.23.0
+        The ``date_created`` property.
```

Comment:
![Screen Shot 2023-04-26 at 10 38 12 AM](https://user-images.githubusercontent.com/96442646/234657815-e82d73bc-ec7a-42b9-a562-4ef7a7f0ef84.png)

Reply:
I would've put the `date_created` one with the actual property, but I don't like this blank space we end up rendering

![Screen Shot 2023-04-26 at 10 29 46 AM](https://user-images.githubusercontent.com/96442646/234657937-2bbda510-6555-4cfb-9993-fff866e4a5c3.png)